R/create_package.R

```diff
@@ -1,9 +1,18 @@
 #' Create an R package according to INBO requirements
+#'
+#' Creates a package template in a new folder.
+#' Use this function when you want to start a new package.
+#' DO READ `vignette("getting_started")` before you run this function.
+#'
+#' @template checklist_structure
+#'
 #' @param package Name of the new package.
 #' @param path Where to create the package directory.
 #' @param title A single sentence with the title of the package.
 #' @param description A single paragraph describing the package.
-#' @param maintainer The output of `orcid2person()`.
+#' @param maintainer The output of [utils::person()] or [`orcid2person()`].
+#'   If you use [utils::person()], then you must provide `given`, `family`,
+#'   `role`, `email` and  `comment` with valid `ORCID`.
```

Comment:
As this part is not covered in the vignette, it certainly needs an example for clarification.  I remember spending quite some time figuring this out, and at first view I see no explanation for it.  (The vignette refers to this function documentation for additional explanation...)

Reply:
example in 2a2283f6e02a72f882b6a0656b27c7e548a8a8a8